nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,83 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals");
+const reservations = require("./data/reservations");
+const reviews = require("./data/reviews");
+
+// this is where you will be adding your routes
+app.get("/", async (request, response) => {
+  response.send("Application Result");
```

Comment:
Hey @hwaleedali 👋 The text you're sending back here ("Application Result") doesn't match with the text you're comparing it to in the routes.spec.js file. Should be an easy fix though.